### ISSUE TITLE
Question of gatord and DS5

### ISSUE THREAD

**Issue by deco-dev** [ARM-software/gator#53]
I want to profile ARM PMUs using DS5 and gatord.
Can I see profiling visualization to realtime in DS5?

**Comment by deco-dev** [comment]
ARM PMUs are CPU PMU, CMN PMU, NI PMU, Mali PMU, etc.

**Comment by bengaineyarm** [comment]
Yes, use Streamline to visualize the output of gator. Nowardays you don't need DS5, use Arm Performance Studio download from https://developer.arm.com/Tools%20and%20Software/Arm%20Performance%20Studio
See the links on that page for more details about Streamline

**Comment by deco-dev** [comment]
Thank you. Streamline just shows hw pmu event? other data does not show like ftrace, proc, meminfo, etc..?

**Comment by deco-dev** [comment]
And streamline is free? no license?
